cmd/config/internal/commands/cmdcreatesubstitution.go

```diff
@@ -64,16 +63,20 @@ func (r *CreateSubstitutionRunner) preRunE(c *cobra.Command, args []string) erro
 		return err
 	}
 
-	// parse the marker values
-	for i := range r.Values {
-		parts := strings.SplitN(r.Values[i], "=", 2)
-		if len(parts) < 2 {
-			return errors.Errorf("values must be specified as PATTERN_MARKER=SETTER_NAME")
-		}
-		ref := setters2.DefinitionsPrefix + setters2.SetterDefinitionPrefix + parts[1]
+	// extract setter name tokens from pattern enclosed in ${}
+	re := regexp.MustCompile(`\$\{([^}]*)\}`)
```

Comment:
Using `${}` may make people think of shell expansion, and using `$()` may make people think of the kubernetes downward api and/or kustomize vars.  

Might as well stick with `${}`; the context seems pretty clear in KptFile comments.

Reply:
Yes. I updated the example with escape for $. $() or $[] are equally bad and need escapes. Do you think there is a better way to delimit avoiding the $?

Reply:
I think people can handle `${}` in this context.
It's tried and true, and people know how to escape it.